.github/workflows/check-merge-conflict.yml

```diff
@@ -0,0 +1,16 @@
+name: Merge Conflict Check
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
```

Comment:
The docs are a bit sparse on this: Does `synchronize` also trigger when a PR is first created? I would think so, right? :thinking: But if not, we probably want another trigger for that event.

Reply:
I don't think it is but GitHub will tell you that your pull request has conflict while you are creating it. Do we want a bot that immediately tells you the same thing?

If so, we probably want to set:

```yml
  pull_request_target:
    types:
      - opened
      - synchronize
```


The events [should be triggered](https://frontside.com/blog/2020-05-26-github-actions-pull_request/#how-do-workflows-trigger-on-pull_request) when…
- opened: pull request is created
- synchronize: commit(s) pushed to the pull request

Reply:
You have a good point there, maybe we don't need it. You decide, I'm fine either way. :)

Reply:
It's now `opened` and `synchronize`